client/js/monthData.js

```diff
@@ -1,151 +1,48 @@
 /*Temporary Mock Month Data */
```

Comment:
is it being used as defaults somewhere? I'd imagine that if our server was down, we'd want to display an error message and ask users to check back later or something